lab11.md

```diff
@@ -0,0 +1,185 @@
+# Lab 11 - Advanced feature - Trust with HMAC
+
+<img src="https://github.com/openfaas/media/raw/master/OpenFaaS_Magnet_3_1_png.png" width="500px"></img>
+
+Before starting this lab create new folder
+```
+mkdir -p lab11 \
+   && cd lab11
+```
+
+## What is HMAC
+
+Currently our functions can get information from everywhere without any restriction. If we want to validate the source of information we can use Hash-based Message Authentication Code (HMAC).
```

Comment:
I'm still not sure about this first sentence.  It could lead people to think their instance is somehow open to the world regardless of any measures they might have in place - NACLs, SGs, Firewalls, etc.

How does this read?

Currently there are no limitations on who can make use of our functions.  As long as connectivity allows, our functions can be invoked without any restriction. By default they trust any supplied information to be genuine.  However, if we want to control access to functions we can use Hash-based Message Authentication Code (HMAC) to validate the source of information.

Reply:
Yeah yours sounds way better 😄